stylix/opacity.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, config, lib, ... } @ args:
+
+with lib;
+
+let
+  cfg = config.stylix.opacity;
+  #fromOs = import ./fromos.nix { inherit lib args; };
```

Comment:
You should use fromOS in defaults to make sure values from NixOS are used in home-manager.